.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -89,6 +93,12 @@ for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
         label_suffix=" (independent agent)"
     fi
     package_name=$(basename "${package}")
+
+    if [[ "$independent_agent" == "false" && "$package_name" == "oracle" ]]; then
+        echoerr "Package \"${package_name}\" skipped: not supported with Elastic Agent running in the stack (missing required software)."
+        continue
+    fi
```

Comment:
Required exception, since CI tests this package with both:
- Generic Elastic stack started with `elastic-package stack up`
- Independent Elastic Agents (setting env. variable)

Elastic Agent from the stack does not have the required libraries for this package.